thoth/report_processing/components/adviser.py

```diff
@@ -29,7 +29,7 @@
 import numpy as np
 
 from thoth.report_processing.exceptions import ThothMissingDatasetAtPath
-
+from thoth.report_processing.utils import map_os_name, normalize_os_version
```

Comment:
You can use methods from thoth-common so we have them all at one place:

```suggestion
from thoth.common import normalize_os_version
from thoth.common import map_os_name
```